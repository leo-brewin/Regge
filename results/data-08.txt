# defect
-3.432541952483177980E-02

# i  j   d(defect)/d(lsq(i,j))
  1  2 -2.464916985827382020E+01
  1  3 -2.478922129047151230E+01
  2  3 -2.467789616064592670E+01
  4  1  3.048339003100295860E+01
  4  2  3.035919543141157350E+01
  4  3  3.051951224297038350E+01
  5  1  3.262955570886601350E+00
  5  2  3.253677529940668810E+00
  5  3  3.267097658253211500E+00
  4  5 -2.094266924040265960E+01
  6  1  3.288178422364719620E+00
  6  2  3.244290320781230880E+00
  6  3  3.290585379608571110E+00
  5  6 -2.089149844377622080E+01
  7  1  3.270036703613657710E+00
  7  2  3.251659108392221600E+00
  7  3  3.273474292518891280E+00
  6  7 -2.089261543890449210E+01
  8  1  3.243420307514030210E+00
  8  2  3.278863640519153460E+00
  8  3  3.249903783980756510E+00
  7  8 -2.095238726961954500E+01
  9  1  3.065076960133482910E+01
  9  2  3.047774591225114330E+01
  9  3  3.068466052347688460E+01
  8  9 -2.098450529068234410E+01
  9  4 -4.442936289110224560E+01
