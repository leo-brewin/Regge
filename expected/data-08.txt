# defect
-3.432541952483177160E-02

# i  j   d(defect)/d(lsq(i,j))
  1  2 -2.464916985827381930E+01
  1  3 -2.478922129047151150E+01
  2  3 -2.467789616064592580E+01
  4  1  3.048339003100295490E+01
  4  2  3.035919543141157220E+01
  4  3  3.051951224297038230E+01
  5  1  3.262955570886601340E+00
  5  2  3.253677529940668820E+00
  5  3  3.267097658253211520E+00
  4  5 -2.094266924040265960E+01
  6  1  3.288178422364719670E+00
  6  2  3.244290320781230890E+00
  6  3  3.290585379608571110E+00
  5  6 -2.089149844377622080E+01
  7  1  3.270036703613657720E+00
  7  2  3.251659108392221590E+00
  7  3  3.273474292518891290E+00
  6  7 -2.089261543890449210E+01
  8  1  3.243420307514030040E+00
  8  2  3.278863640519153450E+00
  8  3  3.249903783980756530E+00
  7  8 -2.095238726961954500E+01
  9  1  3.065076960133482710E+01
  9  2  3.047774591225114200E+01
  9  3  3.068466052347688340E+01
  8  9 -2.098450529068234410E+01
  9  4 -4.442936289110224380E+01
